README.md

```diff
@@ -66,3 +66,9 @@ No sensors yet...but pull requests welcome!
 
 See https://stackstorm.com/2016/10/21/auto-remediation-stackstorm-splunk/ for an example of how to submit events from Splunk
 into StackStorm
+
+## Maintainers
+
+Active pack maintainers with review & write repository access and expertise with Splunk:
+
+* Sean Elliott ([@satellite-no](https://github.com/satellite-no)), deepwatch
```

Comment:
Can you include an email address as well? So users or TSC members would be able to contact the pack maintainer.

`<username>@users.noreply.github.com` would work fine, if you'd prefer privacy.

Reply:
For sure